translations/de/hints/bryukh.html

```diff
@@ -0,0 +1,51 @@
+<body data-author="bryukh">
+<div class="question-answer">
+    <div class="question">
+        Ich habe keine Idee, wie ich anfangen soll, diese Mission zu lösen
+    </div>
+    <div class="answer">
+        Listen in Python haben eine nützliche Methode für diese Mission. --
+        Python lists have an useful method for this mission. --
+        <a href="https://docs.python.org/3/tutorial/datastructures.html#more-on-lists">count(element)</a>.
+    </div>
+</div>
+<div class="question-answer">
+    <div class="question">
+        Ich benötige etwas Hilfe, um mit der Mission weiterfahren zu können
+    </div>
+    <div class="answer">
+        Entferne keine Elemente von der Liste über die Du iterierst, dies kann unvorhersagbare Resultate zur Folge haben.
+        Und, vergiss nicht, dass Deine Funktion das Resultat zurückgeben soll, nicht ausgeben.
+    </div>
+</div>
+<div class="question-answer">
+    <div class="question">
+        Ich bin auf halbem Wege verloren. Ich brauche Hilfe!
+    </div>
+    <div class="answer">
+        Du kannst eine neue leere Liste erzeugen, beispielsweise so: resultat = [], und nicht-einzigartige Elemente hinzufügen.
+    </div>
+</div>
+<div class="question-answer">
+    <div class="question">
+        Ich komme nicht mehr weiter. Ich brauche einen kleinen Hinweis.
+    </div>
+    <div class="answer">
+        "for el in array:", "data.count" und "resultat.append" sind alles was Du für diese Mission benötigst.
+    </div>
+</div>
+<div class="question-answer">
+    <div class="question">
+        Ich habe alles versucht. Nichts geht. SOS.
+    </div>
+    <div class="answer">
+        Versuche einfach die Methode und Anweisungen vom letzten Hinweis.
```

Comment:
to: _Versuche `es einfach mit den Methoden` und Anweisungen `aus dem` letzten Hinweis._